xarray/backends/hdf5.py

```diff
@@ -0,0 +1,190 @@
+from __future__ import absolute_import
```

Comment:
TODO - didn't do anything useful here yet, besides copy-pasting from h5netcdf_.py and renaming variables